packages/theme-chalk/src/common/var.scss

```diff
@@ -773,8 +773,8 @@ $table: map.merge(
     'bg-color': getCssVar('fill-color', 'blank'),
     'tr-bg-color': getCssVar('fill-color', 'blank'),
     'expanded-cell-bg-color': getCssVar('fill-color', 'blank'),
-    'fixed-left-column': inset 10px 0 10px -10px rgb(0 0 0 / 15%),
-    'fixed-right-column': inset -10px 0 10px -10px rgb(0 0 0 / 15%),
+    'fixed-left-column': inset 12px 0 10px -16px rgb(0 0 0 / 50%),
+    'fixed-right-column': inset -12px 0 10px -12px rgb(0 0 0 / 15%),
```

Comment:
I feel like this change doesn't solve the problem

Reply:
Although not elegant, the style issue has indeed been resolved~

Reply:
Can the values of `12px 0 10px -16px rgb(0 0 0 / 50%)`  ​​on both sides be consistent?

Reply:
> Can the values of `12px 0 10px -16px rgb(0 0 0 / 50%)` ​​on both sides be consistent?

It cannot be consistent, it can only be as close as possible, visually consistent.

Reply:
IMO, if we change it like this, it will do more harm than good.

This style difference will affect all users